### PR TITLE
first changes on MVC

### PR DESCRIPTION
**Pull request by premaseem**
Made changes for a banking site, during git training.

### CODE REVIEW

File: MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java [hunk 1]
```diff
@@ -16,8 +16,9 @@ public Flight(AtcMediator atcMediator, String flightName) {
 
 	void grantLandingPermission(){
 		atcMediator.grantLandingPermission(this);
+		System.out.println("just added this line for training")
```
Comment (premaseem) [inline MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java:19]: Please remove null from here and rename the values

File: MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java [hunk 2]
```diff
@@ -3,11 +3,11 @@
 import com.premaseem.mediator.atc.AtcMediator.FlightStatus;
 
 public class Flight {
-	
+
 	FlightStatus status;
 	AtcMediator atcMediator=null;
 	public String name;
-	
+
```
Comment (Nari-P) [inline MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java:10]: These are the variables

File: MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java [hunk 3]
```diff
@@ -3,11 +3,11 @@
 import com.premaseem.mediator.atc.AtcMediator.FlightStatus;
 
 public class Flight {
-	
+
```
Comment (ghost) [inline MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java:6]: finding out pull requests

Comment (premaseem) [reply MediatorPatternExamples/src/com/premaseem/mediator/atc/Flight.java:6]: @Sphoorthi-G done with the suggested changes.

**Review by premaseem — COMMENTED** [review]
Looks good. 
Can be merged.

**Pull request merged**